tests/adapters/switches/remote_test.py

```diff
@@ -548,6 +548,42 @@ def test_get_vlans(self):
         assert_that(vrrp_group2.ips, is_([IPAddress("3.3.3.1")]))
         assert_that(vrrp_group2.priority, is_(100))
 
+    def test_get_interface(self):
+        self.requests_mock.should_receive("get").once().with_args(
+                url=self.netman_url+'/switches/toto/interfaces/ethernet 1/4',
+                headers=self.headers
+        ).and_return(
+                Reply(
+                        content=open_fixture('get_switch_hostname_interface.json').read(),
+                        status_code=200))
+
+        interface = self.switch.get_interface('ethernet 1/4')
+
+        assert_that(interface.name, equal_to("ethernet 1/4"))
+        assert_that(interface.shutdown, equal_to(False))
+        assert_that(interface.port_mode, equal_to(TRUNK))
+        assert_that(interface.access_vlan, equal_to(None))
+        assert_that(interface.trunk_native_vlan, equal_to(2999))
+        assert_that(interface.trunk_vlans, equal_to([3000, 3001, 3002]))
+
+    def test_get_nonexistent_interface(self):
```

Comment:
minor: Would be good to specify what the expectation is for this test. "test_get_nonexistent_interface_raises"